snuba/subscriptions/subscription.py

```diff
@@ -27,39 +27,16 @@ def __init__(self, dataset: Dataset):
         self.dataset = dataset
 
     def create(
-        self,
-        project_id: int,
-        conditions: Sequence[Condition],
-        aggregations: Sequence[Aggregation],
-        time_window: timedelta,
-        resolution: timedelta,
-        timer: Timer,
-    ) -> Tuple[str, str]:
-        if time_window < timedelta(minutes=1):
-            raise InvalidSubscriptionError(
-                "Time window must be greater than or equal to 1 minute"
-            )
-        if resolution < timedelta(minutes=1):
-            raise InvalidSubscriptionError(
-                "Resolution must be greater than or equal to 1 minute"
-            )
-
-        subscription = Subscription(
-            str(uuid1().hex),
-            project_id,
-            conditions,
-            aggregations,
-            time_window,
-            resolution,
-        )
-
+        self, subscription: Subscription, timer: Timer
+    ) -> SubscriptionIdentifier:
         # We want to test the query out here to make sure it's valid and can run
-        request = subscription.build_request(
-            self.dataset, datetime.now() - subscription.time_window, None, timer,
-        )
+        request = subscription.build_request(self.dataset, datetime.now(), None, timer,)
```

Comment:
I think `datetime.now()` might need to be `datetime.utcnow()` for consistency here with code on the write path. Shouldn't matter in production, but local time might yield unexpected results in development. (Don't get me started about tzinfo.)